appveyor.yml

```diff
@@ -1,7 +1,11 @@
 version: "{build}"
-
 os: Visual Studio 2015
 
+branches:
```

Comment:
Will this prevent the double builds when requesting a PR?

Reply:
Yes, that's the idea!